alcs-api/src/providers/typeorm/orm.config.ts

```diff
@@ -15,7 +15,7 @@ export const getTypeOrmModuleOptions = (
   entities: [join(__dirname, '**', '*.entity.{ts,js}')],
   /* Note : it is unsafe to use synchronize: true for schema synchronization
```

Comment:
Can probably remove this comment as well.

Reply:
Removed